doc/Command-Reference.md

```diff
@@ -31,6 +31,9 @@
 * [BGP](#bgp)
   * [BGP show commands](#bgp-show-commands)
   * [BGP config commands](#bgp-config-commands)
+* [Component Firmware](#component-firmware)
+  * [Component Firmware show commands](#component-firmware-show-commands)
+  * [Component Firmware config command](#component-firmware-config-command)
```

Comment:
you are missing s in the config command. All the rest has 'config commands'

Reply:
@liat-grozovik fixed